firefoxThemes/resources.js

```diff
@@ -0,0 +1,66 @@
+/*Location to retrieve resources for each waifu theme*/
+class WaifuThemes{
+    constructor(){
+        this.initThemes();
+    }
+    /*Initialize Waifu Themes*/
+    initThemes(){
+        this.themes = {
+			aquaDark: new Theme("Aqua","waifus/aqua/aqua.png","waifus/aqua/theme.json"),
```

Comment:
So I was expecting to see changes in the `buildSrc/BuildThemes.ts` and I didn't see any.

So I assume you have been manually adding these :scream: 
If so, thank you for all your hard work!

I also don't expect you to be able to make those change either, as you don't have the proper access to 

https://github.com/doki-theme/doki-theme-chrome/blob/d2917a5f40af099e9a892e886a999356273c31fe/buildSrc/src/BuildThemes.ts#L639

I do not want to manually mantain the themes, so I'll work on getting that set up.

Reply:
Yes I have been manually adding them one by one.